src/server/scripts/World/world_script_loader.cpp

```diff
@@ -56,4 +57,5 @@ void AddWorldScripts()
     if (sWorld->getBoolConfig(CONFIG_IP_BASED_ACTION_LOGGING))
         AddSC_action_ip_logger(); // location: scripts\World\action_ip_logger.cpp
     AddSC_duel_reset();
+    AddSC_xp_boost();
```

Comment:
maybe add the script only if sWorld->getIntConfig(CONFIG_XP_BOOST_DAYMASK) != 0 to avoid calling it every time even if disabled ?

Reply:
Hmm, that would disable the ability to use `.reload config` to enable and disable the script though.

Reply:
well it seems like 2 lines above that happens already with if (sWorld->getBoolConfig(CONFIG_IP_BASED_ACTION_LOGGING))

Reply:
Could a WorldScript (afaik that was the name) be used to hook reload config event and enable/disable script on demand?

Reply:
No, there is currently no way to disable a single script